apps/music/js/ui/title_bar.js

```diff
@@ -43,6 +59,17 @@ var TitleBar = {
     ModeManager.pop();
   },
 
+  showScanProgress: function(info) {
+    this.scanProgress.classList.remove('hidden');
+    this.scanCount.textContent = info.count;
+    this.scanArtist.textContent = info.artist || '';
+    this.scanTitle.textContent = info.title || '';
```

Comment:
Okay, make sense, let's keep what it is since this should be a refactoring work.